recipe/meta.yaml

```diff
@@ -45,3 +45,4 @@ extra:
     recipe-maintainers:
       - grlee77
       - patricksnape
+      - jor-
```

Comment:
I don't know how our automation will respond to the punctuation. So I've started up the feedstock update service to see how it goes.